src/Shared/DC/ZRDB/dtml/test.dtml

```diff
@@ -232,4 +226,14 @@
 
 </main>
 
+<dtml-if "REQUEST.get('query_start',None)">
+  <script>
+    //<!--
+    document.addEventListener("DOMContentLoaded", function(event) { 
+      document.location.href="#query_results";
+    });
+    //-->
+  </script>
+</dtml-if>
+
```

Comment:
This is for making the page jump to the start of the search results when the search form has been submitted, right? I tries this on Chromium, Safari and FF on macOS and it didn't jump there. Even if I had written the SQL with so many line breaks that the results were below the viewable area.

Reply:
yes, works perfect in Win-Chrome/FF and makes it easier to walk through the result table. I will ask my Mac-colleagues to propose an alternative JS code ...

Reply:
This snippet on a test page also worked fine for me on a Mac. Example used:

```#html
<html>
<body>
    <div style="height: 500px; background-color: red">
    </div>
    <div style="height: 500px; background-color: red">
    </div>
    <h2 id=fnord>Fnord</h2>
    <div style="height: 500px; background-color: red">
    </div>
    <div style="height: 500px; background-color: red">
    </div>
    <script>
        document.addEventListener("DOMContentLoaded", function(event) { 
              document.location.href="#fnord";
        });
    </script>
</body>
</html>
```

So, it should work.